openfisca_france/model/prelevements_obligatoires/prelevements_sociaux/taxes_salaires_main_oeuvre.py

```diff
@@ -87,14 +87,16 @@ def function(self, simulation, period):
         taux = simulation.legislation_at(period.start).cotsoc.contribution_supplementaire_apprentissage
 
         if period.start.year > 2012:
-            taux_contribution = (
-                (effectif_entreprise < 2000) * (ratio_alternants < .01) * taux.moins_2000_moins_1pc_alternants +
-                (effectif_entreprise >= 2000) * (ratio_alternants < .01) * taux.plus_2000_moins_1pc_alternants +
-                (.01 <= ratio_alternants) * (ratio_alternants < .02) * taux.entre_1_2_pc_alternants +
-                (.02 <= ratio_alternants) * (ratio_alternants < .03) * taux.entre_2_3_pc_alternants +
-                (.03 <= ratio_alternants) * (ratio_alternants < .04) * taux.entre_3_4_pc_alternants +
-                (.04 <= ratio_alternants) * (ratio_alternants < .05) * taux.entre_4_5_pc_alternants
+            taxe_due = (effectif_entreprise >= 250) * (ratio_alternants < .05)
+            taux_conditionnel = (
```

Comment:
`numpy.select` te permet d'avoir une structure avec moins de redondance.

`select([cond1, cond2, cond3], [x1, x2, x3])` renvoit:
-  `x1` si `cond1 == True`
-  `x2` si `cond1 == False and cond2 == True`
-  `x3` si `cond1 == False and cond2 == False and cond3 == True`.